Packs/Endace/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Endace",
+    "description": "Endace",
+    "support": "partner",
+    "serverMinVersion": "5.0.0",
+    "currentVersion": "1.0.0",
+    "author": "Endace",
+    "url": "https://www.endace.com/",
+    "email": "soumen.chakraborty@endace.com",
```

Comment:
note that this email address will be used for support in case users need assistance, are you sure you want to use your personal email address for that?